front/app/containers/Authentication/steps/ClaveUnicaEmail/index.tsx

```diff
@@ -50,7 +50,11 @@ const ClaveUnicaEmail = ({ loading, setError, onSubmit }: Props) => {
         return;
       }
 
-      setError('account_creation_failed');
+      if (e.email?.[0]?.error === 'taken') {
```

Comment:
It looks quite ugly, but I'm not sure how to do it better.

Reply:
I think it's fine, we can improve stuff like this when it's platform week and we improve error handling in general on the platform